doc/guide/install.rst

```diff
@@ -8,25 +8,28 @@ The following sections will guide you through the installation of CLIMADA and it
 
     CLIMADA has a complicated set of dependencies that cannot be installed with ``pip`` alone.
     Please follow the installation instructions carefully!
-    We recommend to use `Anaconda`_ for creating a suitable software environment to execute CLIMADA.
+    We recommend to use `Conda`_ for creating a suitable software environment to execute CLIMADA.
 
-All following instructions should work on any operating system (OS) that is supported by `Anaconda`_, including in particular: **Windows**, **macOS**, and **Linux**.
+All following instructions should work on any operating system (OS) that is supported by `Conda`_, including in particular: **Windows**, **macOS**, and **Linux**.
 
-.. note:: When mentioning the terms "terminal" or "command line" in the following, we are referring to the "Terminal" apps on macOS or Linux and the "Anaconda Prompt" on Windows.
+.. hint:: If you need help with the vocabulary used on this page, refer to the :ref:`Glossary <install-glossary>`.
 
 -------------
 Prerequisites
 -------------
 
 * Make sure you are using the **latest version** of your OS. Install any outstanding **updates**.
 * Free up at least 10 GB of **free storage space** on your machine.
-  Anaconda and the CLIMADA dependencies will require around 5 GB of free space, and you will need at least that much additional space for storing the input and output data of CLIMADA.
+  Conda and the CLIMADA dependencies will require around 5 GB of free space, and you will need at least that much additional space for storing the input and output data of CLIMADA.
 * Ensure a **stable internet connection** for the installation procedure.
   All dependencies will be downloaded from the internet.
   Do **not** use a metered, mobile connection!
-* Install `Anaconda`_, following the `installation instructions <https://docs.anaconda.com/anaconda/install/>`_ for your OS.
+* Install the `Conda`_ package manager.
+  We highly recommend you use `Miniforge`_, which includes the potent `Mamba`_ package manager.
```

Comment:
@sarah-hlsn I just noticed after committing your suggestion that we now double the phrase "package manager" for both Conda and Mamba in two consecutive sentences. Isn't this more confusing? 😬 Maybe we can phrase this just a little bit differently? Drawing from https://docs.conda.io:
```suggestion
* Install the `Conda`_ environment management system.
  We highly recommend you use `Miniforge`_, which includes the potent `Mamba`_ package manager.
```

Reply:
Yes this works very well!